src/sqlparser/tests/sqlparser_postgres.rs

```diff
@@ -768,6 +768,55 @@ fn parse_create_function() {
         }
     );
 
+    // Note that there should NOT exist blank space between
+    // function definition and double dollar signs
```

Comment:
I don't understand the NOTE here. Can you give an example / add a failure test?

Reply:
What I'm referring to is like `create function add(INT, INT) returns int language sql as $$ select $1 + $2 $$`, and it's supported at present 🤣, I'll delete the comment.